DT.iOS.DatePickerDialog/DatePickerDialog.cs

```diff
@@ -269,6 +287,46 @@ private void AddButtonsToView(UIView container)
             container.AddSubview(_doneButton);
         }
 
+
+        private void AddBarButtonsToView(UIView container)
+        {
+            var width = container.Bounds.Size.Width;
+
+            var toolbar = new UIToolbar(new CGRect(
+                0,
+                container.Bounds.Size.Height - kDefaultButtonHeight,
+                width,
+                kDefaultButtonHeight))
+            {
+                Translucent = true,
+                Opaque = false,
+                BackgroundColor = UIColor.Clear,
+                
+            };
+            toolbar.SetBackgroundImage(new UIImage(), UIToolbarPosition.Any, UIBarMetrics.Default);
+            toolbar.Layer.CornerRadius = kCornerRadius;
+            toolbar.Layer.MasksToBounds = true;
+
+            var doneButton = new UIBarButtonItem(UIBarButtonSystemItem.Done, ButtonTapped)
+            {
+                Tag = kDoneButtonTag
+            };
+            var cancelButton = _showCancelButton ? new UIBarButtonItem(UIBarButtonSystemItem.Cancel, ButtonTapped) : null;
+            var spacer = new UIBarButtonItem(UIBarButtonSystemItem.FlexibleSpace);
+            spacer.TintColor = UIColor.Red;
```

Comment:
**Optional**: if I'm reading this right https://blog.prototypr.io/designing-a-dark-mode-for-your-ios-app-the-ultimate-guide-6b043303b941 iOS has different red colors for dark/light themes.
If so then I'd move that to Colors where all theming is to keep it more or less aligned with the original Swift project.

![image](https://user-images.githubusercontent.com/1788581/110386046-c8761e80-802d-11eb-8c85-12155edff2f9.png)

Reply:
This is a typo. This line was needed for debug